### ISSUE TITLE
PHP notice: Undefined index: src

### ISSUE THREAD

**Issue by Ibochkarev** [modxcms/revolution#15499]
### Summary
Log error:

```
[2021-03-06 13:34:06] (WARN @ /Users/bochkarev/Sites/bughunt/core/src/Revolution/Sources/modMediaSource.php : 1920) PHP notice: Undefined index: src
```

### Step to reproduce
Set system setting `log_level` to 4

Go to Media, then select Media browser

### Observed behavior
```
[2021-03-06 13:34:06] (WARN @ /Users/bochkarev/Sites/bughunt/core/src/Revolution/Sources/modMediaSource.php : 1920) PHP notice: Undefined index: src
```

The same behavior is observed when creating a new folder in the assets directory (for example)

### Expected behavior
There should be no errors in the log

### Related issue(s)/PR(s)
NA

### Environment
MODX Revolution 3.0.0-alpha3 (git), PHP Version 7.4.12

**Comment by Ibochkarev** [comment]
The warning is also relevant in the released alpha 3

**Comment by Ibochkarev** [comment]
Fixed #15966
